OpenRA.Mods.Common/Traits/Buildings/TransformsIntoLinkClientManager.cs

```diff
@@ -117,52 +109,36 @@ void IResolveOrder.ResolveOrder(Actor self, Order order)
 
 		string IOrderVoice.VoicePhraseForOrder(Actor self, Order order)
 		{
-			if (order.OrderString == "Dock" && CanDockAt(order.Target.Actor, false))
+			if (order.OrderString == "Link" && CanLinkTo(order.Target.Actor, false))
 				return Info.Voice;
-			else if (order.OrderString == "ForceDock" && CanDockAt(order.Target.Actor, true))
+			else if (order.OrderString == "ForceLink" && CanLinkTo(order.Target.Actor, true))
 				return Info.Voice;
 
 			return null;
 		}
 
 		Order IIssueOrder.IssueOrder(Actor self, IOrderTargeter order, in Target target, bool queued)
 		{
-			if (order.OrderID == "Dock" || order.OrderID == "ForceDock")
+			if (order.OrderID == "Link" || order.OrderID == "ForceLink")
 				return new Order(order.OrderID, self, target, queued);
 
 			return null;
 		}
 
-		/// <summary>Clone of <see cref="DockClientManager.DockingPossible(Actor)"/>.</summary>
-		public bool DockingPossible(Actor target, TargetModifiers modifiers)
+		/// <summary>Clone of <see cref="LinkClientManager.LinkingPossible(Actor, bool)"/>.</summary>
```

Comment:
Obvious comment: Is it possible to just call that method (or perhaps exposing a shared helper or some such) rather than merely documenting the duplication here?

Reply:
That's how all `TransformsInto*` traits are made, IIRC they don't extract out cleanly